src/pages/install/windows.md

```diff
@@ -0,0 +1,91 @@
+
+Note: if you can choose between using Cygwin or MSYS2, be advised that Cygwin is slower and has been reported to cause a bit of trouble to some.
+
+</TabItem>
+<TabItem value="win32" label="None of those">
+
+1. First, [pick the version you want to install](/docs).
+   If you want to [use `master`](/docs/master/#master) instead of a release, [follow the corresponding instructions](/install/master).
+2. Follow the "release page" link below "GitHub links", and grab either of the `win32` (for 32-bit Windows) or `win64` (for 64-bit Windows) `.zip` files, near the bottom of the page.
+3. Unzip that file, you should get the `.exe` files alongside a couple of `.dll`s.
+4. Either:
+   - ...put all of the files in a directory, then add it to the `PATH`.
+     This will **permanently** allow you to use RGBDS **from anywhere**.
+
+     <Tabs>
+     <TabItem value="gui" label="Graphically">
+
+       1. Open Control Panel
+       2. Click "User Accounts"
+       3. Click "User Accounts" again
+       4. Click "Change my environment variables"
+       5. Select the "Path" line in the **top** panel
+       6. Click "Edit..."
+       7. Click "Browse...", select the folder the files are in, and click OK
+       8. Make sure that the new entry (which should be highlighted) is at the bottom of the list; if not, click on "Move Down" until it is
+       9. Click "OK"
+       10. Click "OK"
+
+     </TabItem>
+     <TabItem value="cmd.exe" label="cmd.exe">
+
+       Run the following, replacing `<rgbds_path>` with the path to the directory that contains `rgbasm.exe`, `rgblink.exe`, etc.
+
+       ```cmd
+       setx PATH "%PATH%<rgbds_path>;"
+       ```
+
+       ...then open a new window for the changes to take effect.
+
+       If you only want to modify the PATH temporarily, instead of the permanent [`setx` command](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx), you can use  the **temporary** [`set`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1).
+
+     </TabItem>
+     <TabItem value="pwsh" label="PowerShell">
+
+       Run the following, replacing `<rgbds_path>` with the path to the directory that contains `rgbasm.exe`, `rgblink.exe`, etc.
+
+       ```cmd
+       setx PATH ${Env:PATH}<rgbds_path>;
+       ```
+
+       ...then open a new window for the changes to take effect.
+
+       If you only want to modify the PATH temporarily, instead of the permanent [`setx` command](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx), you can use  the **temporary** [`set`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1).
+
+     </TabItem>
+     </Tabs>
+
+   - ...or put all of the files in your project's directory.
+   - ...or put all of the files in a directory already in the `PATH`.
```

Comment:
```suggestion
   2. ...or put all of the files in a folder that's already in the `PATH`.
   3. ...or put all of the files directly in your project's folder.
```

Reply:
The rest of the list (see above the tabs) is unordered, since it's not meant to be a list of steps; so this shouldn't be either.

Also the "directory" nitpick from above.

Reply:
I intended for the numbers to make individual steps easier to refer to (either when talking about the instructions, or mentally to yourself). Also I think nested lists show up as A/B/C?

Reply:
Yes, but those are not individual steps, they are alternatives ways to perform step 8... so an ordered list is not appropriate semantically.

Yeah, the nested lists show up as that: 
![image](https://github.com/gbdev/rgbds-www/assets/15271137/d9530d01-c163-445d-ab2a-ce1669036904)

Reply:
Okay, I've updated the suggestion to not change the bullets.

Reply:
Okay, it looks okay to me!

That said, how about we add a sentence explaining why this or that method is recommended or not?

Reply:
Yeah, that would be useful.

Reply:
I commented separately re: the recommendation sentences for each method (WSL, Cygwin/MSYS2, or neither). I don't think the PATH-updating methods (GUI, cmd, or pwsh) need such recommendations, since that's a one-time operation and they all give identical results.